modules/core/src/main/scala/org/tessellation/infrastructure/snapshot/GlobalSnapshotStorage.scala

```diff
@@ -140,6 +141,11 @@ object GlobalSnapshotStorage {
         def getLatestBalances: F[Option[Map[Address, Balance]]] =
           head.map(_.map(_.info.balances))
 
+        def getLatestBalancesStream: Stream[F, Map[Address, Balance]] =
+          headRef.discrete
```

Comment:
>  Are you sure that SignallingRef works in the same way as the common Ref?

The docs says:
_The update methods have the same semantics as Ref, as well as propagating changes to discrete (with a last-update-wins policy in case of very fast updates)._

I agree with making a test with multiple subscribers, it will be useful for the future 👍

Reply:
I have added two test cases in `GlobalSnapshotStorageSuite`. Please take a look at them and tell me what do you think about them.